backend/backend.config.js

```diff
@@ -3,7 +3,7 @@ require('dotenv').config();
 
 module.exports = {
   substrateNetwork: process.env.SUBSTRATE_NETWORK || 'kusama',
-  wsProviderUrl: process.env.WS_PROVIDER_URL || 'wss://rpc.v2.devnet.cere.network/ws',
+  wsProviderUrl: process.env.WS_PROVIDER_URL || 'wss://archive.devnet.cere.network/ws',
```

Comment:
Revert to rpc

Reply:
I thought that for cerestats we should update all to archive. Fixed.

Reply:
If you look into [k8 configuration ](https://github.com/cere-io/infra-k8s-deployment/blob/286bee57ae18e24b30f49d00a6dd0a43be4d26a4/cerestats/cerestats-crawler/envs/dev.yaml#L9) then see  that it has  `archive`. So why `cerestats` should have different logic?